imqsauth/http.go

```diff
@@ -373,6 +375,55 @@ func httpHandlerCreateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	}
 }
 
+func httpHandlerCreateGroup(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+	groupname := strings.TrimSpace(r.http.URL.Query().Get("groupname"))
+	if groupname == "" {
+		authaus.HttpSendTxt(w, http.StatusNotAcceptable, "Group name may not be blank.")
+		return
+	}
+
+	if _, error := authaus.LoadOrCreateGroup(central.Central, groupname, true); error == nil {
+		central.Central.Log.Printf("New group added: '" + groupname + "'")
+		authaus.HttpSendTxt(w, http.StatusOK, "Group created ('"+groupname+"')")
+		return
+	} else {
+		central.Central.Log.Printf("Error creating group ('" + groupname + "') ")
+		authaus.HttpSendTxt(w, http.StatusBadRequest, "Error creating group ('"+groupname+"') ")
+		return
+	}
+}
+
+func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+	//TODO : Add check so current user cannot remove own Admin rights
+	groupname := strings.TrimSpace(r.http.URL.Query().Get("groupname"))
+	rolesstring := strings.TrimSpace(r.http.URL.Query().Get("roles"))
+
+	roles := strings.Split(rolesstring, ",")
+
+	perms := []authaus.PermissionU16{}
+	permsString := []string{}
```

Comment:
I'm pretty sure you can get rid of permsString. For the log message, just output the original rolesString.